Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -65,15 +69,20 @@ import Foundation
         }
         
         super.init()
+
         BTAPIClient._analyticsService = BTAnalyticsService(apiClient: self)
         http?.networkTimingDelegate = self
 
         // Kickoff the background request to fetch the config
         fetchOrReturnRemoteConfiguration { configuration, error in
```

Comment:
I think we are deciding to disregard prefetch - this would simplify the implementation.

Reply:
Would this save time though for some merchant integrations? Do we want to make sure that BTAnalyticsService is instantiated here? If so, how should we handle the injection. I wrote updateConfiguration function here but also thinking about how merchant would integrate DataCollector which also uses fetchOrReturnRemoteConfiguration.

Reply:
I wonder how much time we are saving the merchant. If Android `v5` doesn't have a prefetch, I think we could follow the same approach. If our concern is to always have `BTAnalyticsService` updated with the configuration values, we could implement an observable pattern

Reply:
Yes, I think it's hard to measure except to assume that there won't be a delay for some merchants' integrations where they'd already have config from v1/configurations kicked off in the background and cached.

So for the merchant integration that instantiates BTAPIClient and payment clients earlier before tokenize functions, they'd reduce latency by typical duration of the v1/configuration call when they call functions like tokenize.